qa/Rakefile

```diff
@@ -18,36 +18,47 @@ namespace :test do
     end
   end
 
-  desc "Bootstrap all the VM's used for this tests"
-  task "setup" do
-    puts "bootstraping all VM's defined in acceptance/Vagrantfile"
-    cd "acceptance" do
-      LogStash::VagrantHelpers.bootstrap
-    end
-  end
-
   namespace :acceptance do
 
     desc "Run all acceptance"
     task :all do
-      exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/**/*_spec.rb"]]))
+      all_boxes = LogStash::VagrantHelpers::DEFAULT_CENTOS_BOXES + LogStash::VagrantHelpers::DEFAULT_DEBIAN_BOXES 
+      for box in all_boxes do
+        LogStash::VagrantHelpers.bootstrap(box)
+        exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/**/*_spec.rb"]]))
+        LogStash::VagrantHelpers.destroy(box)
```

Comment:
I guess the best approach here would be to halt each vm after the run, and then make sure that before we start any new run we do with a clean VM, what do you think?